test/cookbooks/osl-matrix-test/recipes/synapse.rb

```diff
@@ -0,0 +1,25 @@
+# Create the synapse docker container
+osl_synapse 'chat.osuosl.intnet' do
+  app_services %w(osl-irc-bridge)
+  use_sqlite true
+  config <<~EOF
+modules:
+  - module: "ldap_auth_provider.LdapAuthProviderModule"
+    config:
+      enabled: true
+      uri: "ldap://ldap.osuosl.org:389"
+      start_tls: true
+      base: "ou=People,dc=osuosl,dc=org"
+      attributes:
+        uid: "uid"
+        mail: "mail"
+        name: "givenName"
+  EOF
```

Comment:
Why can't this be given as a Hash since it just seems to be? If this needs to be converted to yaml, there is a ruby library which will do that for you.